scripts/generate_certs.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+"""
+This script updates tests/src/certs.c using data from data_files.
+
+You must run this script from an Mbed TLS root.
+
+"""
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import os
+import re
+import subprocess
+import tempfile
+
+class UpdaterCerts:
```

Comment:
Nit: CertUpdater would probably be more natural

Reply:
Thank for suggestion